io.gitlab.leesonwai.Tactics.json

```diff
@@ -0,0 +1,41 @@
+{
+  "app-id" : "io.gitlab.leesonwai.Tactics",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "tactics",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland"
```

Comment:
you don't need GL? since it's a Gtk4 app I'd expect to

Reply:
Thanks for making me aware :smile:.